pkgs/by-name/sy/systemd-lsp/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "systemd-lsp";
+  version = "2025-06-25";
```

```suggestion
  version = "0-unstable-2025-06-25";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning